tests/Util/SortedListTest.php

```diff
@@ -1,56 +0,0 @@
-<?php
-/**
- * @copyright Zicht Online <https://zicht.nl>
- */
-
-namespace ZichtTest\Bundle\FrameworkExtraBundle\Util;
-
-use PHPUnit\Framework\TestCase;
-use Zicht\Bundle\FrameworkExtraBundle\Util\SortedList;
-
-class SortedListTest extends TestCase
-{
-    /**
-     * Test handling of priorities by SortedList.
-     */
-    public function testSortedListPriority()
-    {
-        $s = new SortedList();
```

Comment:
We zouden ook de `src/Util/Sorted*` code kunnen verwijderen, deze is (highly) deprecated en ook niet meer in gebruik zo te zien https://github.com/search?q=org%3Azicht+sortedlist&type=code | https://github.com/zicht/url-bundle/pull/34/files#diff-9245335af62e5fd8b576f6b0699114092628c2af87579faf544d388e699a3034L11

Reply:
Wat mij betreft prima :+1:

Reply:
Mee eens. Het is ook behoorlijk overbodig. Een usort() is zo geschreven. Zeker met de short function notation.

Reply:
Done